app/pages/reporter/view-application.tsx

```diff
@@ -59,7 +77,54 @@ class ViewApplication extends Component<Props> {
   render() {
     const {session} = this.props.query;
     const {query} = this.props;
-    const formResults = query.application.orderedFormResults.edges;
+    const reviewComments = query.application?.reviewCommentsByApplicationId.edges.map(
+      (result) => result.node.description
+    );
+    const status =
+      query?.application?.applicationRevisionStatus?.applicationRevisionStatus;
+    const changesRequested = status === 'REQUESTED_CHANGES';
+    const hasBeenReviewed = status !== 'SUBMITTED' && status !== 'DRAFT';
+
+    const thisVersion = Number(this.props.router.query.version);
+    const latestSubmittedRevision =
+      query.application.latestSubmittedRevision.versionNumber;
+    const latestDraftRevision =
+      query.application.latestDraftRevision.versionNumber;
```

Comment:
```suggestion
      query.application.latestDraftRevision?.versionNumber;
```

latestSubmittedRevision and latestDraftRevision can be null and need optional chaining here. This is what is breaking the cypress test.